ecs-logs-collector.sh

```diff
@@ -419,19 +418,20 @@ get_docker_info()
 {
   try "gather Docker daemon information"
 
-  pgrep docker > /dev/null
+  ps -ef |grep dockerd |grep -v grep  >> /dev/null
   if [[ "$?" -eq 0 ]]; then
     mkdir -p ${info_system}/docker
 
-    docker info > ${info_system}/docker/docker-info.txt 2>&1
-    docker ps --all --no-trunc > ${info_system}/docker/docker-ps.txt 2>&1
-    docker images > ${info_system}/docker/docker-images.txt 2>&1
-    docker version > ${info_system}/docker/docker-version.txt 2>&1
+    timeout 75 docker info > ${info_system}/docker/docker-info.txt 2>&1 || echo "Timed out, ignoring \"docker info output \" "
```

Comment:
Nice. These timeouts may be a little too generous however, is there a reason 75 was chosen? What do you think about reducing these to 10s or lower?

Reply:
@jahkeup it is arbitrary value (just wanted to give enough time before timing out). I can modify it to 20 seconds as I have seen cases where instance will have high CPU utilization which could delay information collection. Does that sound like a decent time ?